.drone.yml

```diff
@@ -7,6 +7,8 @@ pipeline:
     image: ncatelli/golang:1.10.2-alpine3.7
     commands:
       - make
+      - curl https://glide.sh/get | sh
+      - glide init ; glide update ; glide install
```

Comment:
Should probably just make these separate command lines. no need for a multiline.